dime-research-standards/pillar-3-research-reproducibility/checklists/tex/reproducibility-package-guidance-note.tex

```diff
@@ -0,0 +1,109 @@
+		\item \textbf{README.txt:} Text file delineating the requirements of the code and datasets included in the package, instructions about the directory structure DIME Analytics should follow, and any ad-hoc instructions to run the master script.
+		\item \textbf{Filled-in reproducibility check checklist:} DIME Analytics ask submitting teams to provide \href{https://raw.githubusercontent.com/worldbank/dime-standards/master/dime-research-standards/pillar-3-research-reproducibility/checklists/Reproducibility%20package%20submission%20checklist.pdf}{this checklist} so they can make sure they are submitting all the files needed to run a reproducibility package.
+	\end{itemize}
+
+	DIME Analytics will only change the topmost global directory specified in the master script and will run it to reproduce the results – this is what is called a one-button run. The master script code should run without errors after adding the correct folder path of the computer where it will be running. For the paper to be considered fully reproducible, DIME Analytics should reproduce the every exhibit in the paper showing analysis results through this push-button exercise.
+
+	\bigskip
+
+	After receiving a reproducibility package, DIME Analytics will attempt a first one-button run. If the master script breaks, DIME Analytics will identify the point where it is breaking, document it, and attempt to make changes to continue with the review. If the corrections cannot be identified after a 15-minute check, the project team will be notified that further edits are required to conduct the reproducibility check.
+
+	\bigskip
+
+	If the code runs from beginning to end, DIME Analytics will send the team an email confirming that we were able to run the code and providing an estimated time to deliver the results of the review. Reproducibility checks take ten working days (two weeks) in most cases, but might take longer for computationally intensive packages, packages with more than 1 GB of total file size, or when a large volume of submissions is received.
+
+	\bigskip
+	
+	\begin{center}
+		\includegraphics[width=0.9\linewidth]{../img/rep-checks-timeline.png}
+	\end{center}
+
+	\subsection{2. \, Reproducibility check}
+
+	After the one-button run works, DIME Analytics will check the following code requirements. The submission should pass all of these checks to be in compliance with DIME Reproducibility Standards.
+
+	\bigskip
+
+	\begin{itemize}
+		\setlength\itemsep{-0.1em}
+		\item \textbf{Code completeness:} Every paper exhibit containing analysis results should be produced programatically by the code and be a result of statistical analysis. Manual entry of values used in a table or plot, or point-and-click-generated exhibits are not in compliance with DIME Reproducibility Standards.
+		\item \textbf{Output stability:} DIME Analytics will run the code between three to five times. Every output should be  identical across each run.
```

Comment:
we usually also do on multiple machines right? but not necessarily multiple OS?

Reply:
that's correct. We almost always run the results in Windows